Source/WebKit/NetworkProcess/cocoa/WebSocketTaskCocoa.mm

```diff
@@ -91,7 +91,7 @@
         if (message.type == NSURLSessionWebSocketMessageTypeString)
             m_channel.didReceiveText(message.string);
         else
-            m_channel.didReceiveBinaryData(static_cast<const uint8_t*>(message.data.bytes), message.data.length);
+            m_channel.didReceiveBinaryData(std::span { static_cast<const uint8_t*>(message.data.bytes), message.data.length });
```

Comment:
I'll try and land my other PR first so I can update this PR to call `toSpan(message.data)` here.